orchagent/switchorch.cpp

```diff
@@ -107,6 +108,10 @@ void SwitchOrch::doTask(Consumer &consumer)
                         memcpy(attr.value.mac, mac_addr.getMac(), sizeof(sai_mac_t));
                         break;
 
+                   case SAI_SWITCH_ATTR_SWITCH_SHELL_ENABLE:
```

Comment:
in sonic-sairedis, we have startDiagShell function to enable debug shell, have you checked that?